hack/image/push-image-tags.sh

```diff
@@ -1,8 +1,5 @@
 #!/usr/bin/env bash
-
-source hack/lib/common.sh
-
-semver_regex="^v(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)$"
+source hack/lib/image_lib.sh
```

Comment:
I don't see this file in your branch. Perhaps just got lost in the refactoring?

Reply:
Oops. It did. Thanks! I hadn't `git add`-ed it back in.